content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -135,6 +148,102 @@ $ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<t
 <title>Simple Bookstore App</title>
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+Create a [Kubernetes Gateway](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.Gateway),
+so you can access the bookinfo app from outside the cluster:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+  namespace: istio-system
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+{{< /text >}}
```

Comment:
One of the goals of the gateway-api docs, is to try to make the delta between Istio config and Gateway-api be as small as possible so that users don't get the impression that switching APIs is a big deal.

There are already gateway-api yamls for the bookinfo sample that you should reuse in this doc:

```suggestion
{{< text bash >}}
$ kubectl apply -f @samples/bookinfo/gateway-api/bookinfo-gateway.yaml@
{{< /text >}}
```

Reply:
@frankbu thanks for the review and pointer. I'll update and test.

Reply:
@frankbu after further analysis, we can not use `samples/bookinfo/gateway-api/bookinfo-gateway.yaml` without disabling ambient for the Gateway pod due to https://github.com/istio/istio/issues/43587. Let's keep the inline manifest until this issue is fixed.

Reply:
I'm not sure I understand the problem. If it's because the gateway needs to be in a different ns from the other services, won't `kubectl apply -n istio-system -f @samples/bookinfo/gateway-api/bookinfo-gateway.yaml@` work?

Reply:
Until https://github.com/istio/istio/issues/43587 is fixed, Gateways are incapable of terminating HBONE, e.g. running in the ambient mesh. As a workaround, this PR deploys the Gateway in the istio-system namespace (just like the classic Istio ingress gateway) so the client ztunnel > Gateway uses TCP. If the Gateway is deployed in an ambient namespace, then ambient must be disabled for the Gateway pod.

Reply:
That makes sense, but I still don't understand why you said:

> Let's keep the inline manifest until this issue is fixed.

Why can't you use the samples yaml but just install it in the istio-system ns, like I suggested above?

Reply:
OK, I'll do that.

Reply:
IMHO `samples/bookinfo/gateway-api/bookinfo-gateway.yaml` should be updated to only include the Gateway and a new `samples/bookinfo/gateway-api/bookinfo-httproute.yaml` should be introduced for the HTTPRoute. This will provide additional flexibility to work with the samples, e.g. cross namespace routing. Thoughts @frankbu?

Reply:
https://github.com/istio/istio/pull/44101 adds a separate HTTPRoute for bookinfo.

Reply:
- This PR needs to continue inlining the Gateway instead of using `samples/bookinfo/gateway-api` b/c the HTTPRoute in this doc is created in a different ns. This requires the allowed routes `from: All `for the Gateway instead of `from: Same`.
- This PR needs to continue inlining the HTTPRoute instead of using `samples/bookinfo/gateway-api` b/c the HTTPRoute in this doc is created in a different ns. This requires the parentRef to specify `namespace` as the ns of the Gateway (`istio-system`).

Reply:
In that case, I think the new gateway-httproute.yaml that you added in bookinfo samples is not really useful. Should we revert?

I still would prefer using a simple sed script to update `samples/bookinfo/gateway-api/bookinfo-gateway.yaml`, instead of adding all the noise in this doc, since it's really just a temporary fix until the ztunnel issue is fixed.
Here's a version of the command that will add the namespace in both places:
```
sed '/name: bookinfo-gateway/a\
  namespace: istio-system\
' @samples/bookinfo/gateway-api/bookinfo-gateway.yaml@ | kubectl apply -f -
```

Reply:
>In that case, I think the new gateway-httproute.yaml that you added in bookinfo samples is not really useful. Should we revert?

Yes, here's the PR: https://github.com/istio/istio/pull/44172

> I still would prefer using a simple sed script to update samples/bookinfo/gateway-api/bookinfo-gateway.yaml, instead of adding all the noise in this doc, since it's really just a temporary fix until the ztunnel issue is fixed.

This is not a temporary fix. The Gateway in this doc is not only running in a different ns than `samples/bookinfo/gateway-api/bookinfo-gateway.yaml`, the spec differs (allows xRoute binding from all ns's). The `samples/bookinfo/gateway-api/bookinfo-gateway.yaml` also creates an HTTPRoute in the istio-system ns, when the HTTPRoute needs to reside in the default namespace. It seems like the flow of this doc is consistent with the [existing Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/) doc that inlines the Gateway and HTTPRoute.

Reply:
> This is not a temporary fix.

Yes it is. As soon as the ztunnel issue is fixed we will just install both the gateway and route in the default ns by simply applying bookinfo-gateway.yaml. The fact that we can't do that today is a bug.

> It seems like the flow of this doc is consistent with the [existing Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/) doc that inlines the Gateway and HTTPRoute.

The difference is that doc is teaching people about Gateway API so showing them the resources explicitly makes sense. In this doc, the bookinfo Gateway and HTTPRoute are just used for testing, so we don't need/want to dwell on the details here.